packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationProfileRoutes.tsx

```diff
@@ -1,14 +1,14 @@
 import { Protect } from '../../common';
 import { CustomPageContentContainer } from '../../common/CustomPageContentContainer';
-import { useOrganizationProfileContext } from '../../contexts';
+import { useEnvironment, useOrganizationProfileContext } from '../../contexts';
 import { Route, Switch } from '../../router';
 import { OrganizationBilling } from './OrganizationBilling';
 import { OrganizationGeneralPage } from './OrganizationGeneralPage';
 import { OrganizationMembers } from './OrganizationMembers';
 
 export const OrganizationProfileRoutes = () => {
   const { pages, isMembersPageRoot, isGeneralPageRoot, isBillingPageRoot } = useOrganizationProfileContext();
-
+  const { billing } = useEnvironment().organizationSettings;
```

Comment:
Shall we move this logic inside to the `OrganizationBilling` similar to what we are doing to the [Members ](https://github.com/clerk/javascript/blob/e7ae9c36ae66babdad79ce0185b1c6458c08a7b4/packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx#L30) with the useProtect?

Reply:
I think that we need to wrap the route in `OrganizationProfileRoutes` with `Protect` component when we have the new permissions instead of doing this